lib/services/postgresqlManagement/lib/models/index.d.ts

```diff
@@ -47,23 +47,46 @@ export interface TrackedResource extends ProxyResource {
   tags?: { [propertyName: string]: string };
 }
 
+/**
+ * @class
+ * Initializes a new instance of the StorageProfile class.
+ * @constructor
+ * Storage Profile properties of a server
+ *
+ * @member {number} [backupRetentionDays] Backup retention days for the server.
+ * @member {string} [geoRedundantBackup] Enable Geo-redundant or not for server
+ * backup. Possible values include: 'Enabled', 'Disabled'
+ * @member {number} [storageMB] Max storage allowed for a server.
+ */
+export interface StorageProfile {
+  backupRetentionDays?: number;
+  geoRedundantBackup?: string;
+  storageMB?: number;
+}
+
 /**
  * @class
  * Initializes a new instance of the ServerPropertiesForCreate class.
  * @constructor
  * The properties used to create a new server.
  *
- * @member {number} [storageMB] The maximum storage allowed for a server.
  * @member {string} [version] Server version. Possible values include: '9.5',
  * '9.6'
  * @member {string} [sslEnforcement] Enable ssl enforcement or not when connect
  * to server. Possible values include: 'Enabled', 'Disabled'
+ * @member {object} [storageProfile] Storage profile of a server.
+ * @member {number} [storageProfile.backupRetentionDays] Backup retention days
+ * for the server.
+ * @member {string} [storageProfile.geoRedundantBackup] Enable Geo-redundant or
+ * not for server backup. Possible values include: 'Enabled', 'Disabled'
+ * @member {number} [storageProfile.storageMB] Max storage allowed for a
+ * server.
  * @member {string} createMode Polymorphic Discriminator
  */
 export interface ServerPropertiesForCreate {
-  storageMB?: number;
```

Comment:
Removing optional parameter is a breaking change. Requires major version increment.